api_server.py

```
@@ -251,6 +243,83 @@ def check_rules(available_crew: List[Dict]) -> Dict:
+        if not row or not row[0]:
+            return 3  # Default if no data
+
+        max_end_time = row[0]
+        if isinstance(max_end_time, str):
+            max_end_time = datetime.fromisoformat(max_end_time)
+        max_end_time = ensure_london(max_end_time)
+
+        now = get_now()
+        delta = max_end_time - now
+        days = max(1, delta.days)
+        return days
```

Ruff (0.15.9)

[warning] Consider moving this statement to an `else` block

(TRY300)